src/PrestaShopBundle/Entity/AttributeGroup.php

```diff
@@ -181,6 +190,14 @@ public function getPosition()
         return $this->position;
     }
 
+    /**
+     * @return Collection
+     */
+    public function getAttributes(): Collection
```

Comment:
```suggestion
     * @return ArrayCollection
     */
    public function getAttributes(): ArrayCollection
```

Reply:
Nope I tried that but it doesn't work Because Doctrine hydrates the collection into different types, not always ArrayCollection and it crashes, so it's better to use the Collection interface as a type

Reply:
ok.

Reply:
`Type error: Return value of PrestaShopBundle\Entity\AttributeGroup::getAttributes() must be an instance of Doctrine\Common\Collections\ArrayCollection, instance of Doctrine\ORM\PersistentCollection returned (Behat\Testwork\Call\Exception\FatalThrowableError)`